Header header

# Binary npy color image of the scene
string color_data

# Binary npy depth image of the scene
# string depth_data
float32[] depth_data
